packages/react-composites/src/composites/CallComposite/CallComposite.tsx

```diff
@@ -102,6 +102,19 @@ export interface RemoteVideoTileMenuOptions {
    */
   isHidden?: boolean;
 }
+
+/* @conditional-compile-remove(click-to-call) */
+/**
+ * Options for the local video tile in the Call composite.
+ *
+ * @beta
+ */
+export interface LocalVideoTileOptions {
+  /**
+   * Position of the local video tile. If unset will render the local tile in the floating local position.
```

Comment:
When overflow gallery is showing `floating` isn't quite right as its docked to the bottom right